IMDSSample.ps1

```diff
@@ -6,14 +6,14 @@ $NonceValue = "123456"
 function Query-InstanceEndpoint
 {
     $uri = $InstanceEndpoint + "?api-version=2019-03-11"
-    $result = Invoke-RestMethod -Method GET -Uri $uri -Headers @{"Metadata"="True"}
+    $result = Invoke-RestMethod -Method GET -NoProxy -Uri $uri -Headers @{"Metadata"="True"}
```

Comment:
Can we change this to -Proxy $Null as NoProxy doesn't work with powershell 5 and below